packages/core/examples/radioExample.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+import { Radio, RadioGroup } from "@blueprintjs/core";
+import { BaseExample, handleStringChange } from "@blueprintjs/docs";
+
+export interface IRadioExampleState {
+    radioValue?: string;
+}
+
+// tslint:disable-next-line:max-classes-per-file
```

Comment:
should be safe to enable this rule now (remove the line). was only necessary cuz we had multiple things in one file (more proof this PR is correct).